doc/dataframe.md

```diff
@@ -36,7 +36,7 @@ object DataFrameExample {
       .config("spark.redis.port", "6379")
       .getOrCreate()
 
-    val personSeq = Seq(Person("John", 30), Person("Peter", 45))
+    val personSeq = Seq(Person("John", 30), Person("Peter", 45)
```

Comment:
Looks like a typo, shouldn't be any changes on this line

Reply:
Ah I thought I was eliminating a newline of that parenthesis, will fix.